generator/api/template/src/test/test.js

```diff
@@ -3,40 +3,16 @@ import 'babel-polyfill';
 import assert from 'assert';
 import GitHubUser from '../lib/GitHubUser';
 
-// WARNING: don't use arrow functions with mocha describe(), it(),
```

Comment:
The issue I have with it is that it doesn't feel like the right spot for it.  `src/test/test.js` is functioning sample code, great for quick copy-pasta.  The function comment feels like it should be in a tips/ tricks, pitfalls, faq etc . . . document.